### PR TITLE
Add pyreadline recipe

### PR DESCRIPTION
**Pull request by astaric**

### CODE REVIEW

File: recipes/pyreadline/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - pyreadline
```
Comment (synapticarbors) [inline recipes/pyreadline/meta.yaml]: Please add the following additional imports:

```
    - pyreadline.clipboard
    - pyreadline.configuration
    - pyreadline.console
    - pyreadline.keysyms
    - pyreadline.lineeditor
    - pyreadline.modes
```

File: recipes/pyreadline/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+about:
+  home: http://github.com/simplejson/simplejson
```
Comment (synapticarbors) [inline recipes/pyreadline/meta.yaml]: Please change this to: http://ipython.org/pyreadline

**Comment by astaric** [comment]
Thank you for your suggestions, I have pushed the updated version of the recipe.

**Comment by astaric** [comment]
Had to remove pyreadline.configuration from import tests as it is not a real python package (no __init__.py), which cannot be imported in python 2.*. I guess it is treated as a namespace package in python3 so the import worked there.

Or should i just skip the import test for the specific package on py2k?

**Pull request merged**